.github/CONTRIBUTING.md

```diff
@@ -34,7 +34,7 @@ A bug report should include:
 
 ## Suggesting new features
 
-Please use the [discussion forums](http://dotspatial.codeplex.com/discussions) or add it directly in the issue tracker in the same way as bug reports.
+Please use the [discussion forums](https://github.com/DotSpatial/DotSpatial/issues) or add it directly in the issue tracker in the same way as bug reports.
```

Comment:
Change this to discussions as well, and might add the issue link to issue tracker.